migrations/2_deploy.js

```diff
@@ -31,19 +28,19 @@ module.exports = async function(deployer, network, accounts) {
   // Creating DeployerApp helper.
   const deployerApp = new DeployerApp(deployer, web3, deployerAccount, network);
 
-  await deployerApp.deploys([ AddressLib, SafeMath, ZDai ], deployOptions);
+  await deployerApp.deploy(ZDai, deployOptions);
 
   const { chainlink: { dataContracts: { eth_usd } } } = networkConfig;
-  assert(eth_usd, 'Chainlinnk ETH/USD data contract is undefined.');
+  assert(eth_usd, 'Chainlink ETH/USD data contract is undefined.');
   await deployerApp.deploy(EtherUsdAggregator, eth_usd, deployOptions);
 
-  await deployerApp.links(LendingPool, [ AddressLib ]);
   await deployerApp.deploy(LendingPool, deployOptions);
 
-  await deployerApp.links(Lenders, [ AddressLib, SafeMath ]);
-  await deployerApp.deploy(Lenders, ZDai.address, LendingPool.address, deployOptions);
+  await deployerApp.deploy(InterestConsensus, 2, 0, deployOptions);
+  console.log('starting lender info')
 
-  await deployerApp.links(Loans, [ SafeMath ]);
```

Comment:
Why did you remove the link library/contract statement?

Reply:
See above